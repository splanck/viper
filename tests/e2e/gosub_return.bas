10 GOSUB 100
20 PRINT "After first"
30 GOSUB 200
40 PRINT "After second"
50 END
100 PRINT "Enter first"
110 GOSUB 300
120 PRINT "Leave first"
130 RETURN
200 PRINT "Enter second"
210 RETURN
300 PRINT "Enter inner"
310 RETURN
