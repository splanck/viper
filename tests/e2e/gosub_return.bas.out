Enter first
Enter inner
Leave first
After first
Enter second
After second
